### PR TITLE
updated dependencies

### PR DESCRIPTION
**Pull request by istrator2**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -11,17 +11,18 @@ packages = [{include = "extremeflash"}]
 python = "^3.10"
 pyserial = "^3.5"
 tftpy = "^0.8.2"
-paramiko = "^3.0.0"
+paramiko = "^3.3.1"
 scp = "^0.14.5"
+poetry = "^1.7.1"
```
Comment (grische) [inline pyproject.toml:16]: I don't think we want to install poetry inside the poetry venv 😉

**Review by grische — CHANGES_REQUESTED** [review]

**Comment by istrator2** [comment]
fixed poetry inside the poetry venv
downgraded paramico as requested

File: pyproject.toml (hunk 1 above)
Comment (istrator2) [reply pyproject.toml]: fixed poetry inside the poetry venv
downgraded paramico as requested

**Review by grische — APPROVED** [review]
Thanks!

**Pull request merged**